docs/current_docs/developer-guide/go/snippets/services/expose-dagger-services-to-host/main.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"context"
+	"io"
+	"net/http"
+)
+
+type MyModule struct{}
+
+// starts and returns an HTTP service, then exposes it to the host
+func (m *MyModule) HttpService(ctx context.Context) (string, error) {
+	// create HTTP service container with exposed port 8080
+	httpSrv := dag.Container().
+		From("python").
+		WithWorkdir("/srv").
+		WithNewFile("index.html", ContainerWithNewFileOpts{
+			Contents: "Hello, world!",
+		}).
+		WithExec([]string{"python", "-m", "http.server", "8080"}).
+		WithExposedPort(8080).
+		AsService()
+
+	// expose HTTP service to host
+	// FIXME
```

Comment:
This entire snippet needs to be rewritten, or maybe it should be omitted in favour of the example in the new services page. I'm not sure how to rewrite it. Can you help @vito?